saleor/graphql/shipping/mutations/shippings.py

```diff
@@ -305,7 +311,56 @@ def clean_input(cls, info, instance, data, input_cls=None):
                     )
                 }
             )
-        return cleaned_input
+
+    @classmethod
+    def clean_delivery_time(cls, instance, cleaned_input, errors):
+        """Validate delivery days.
+
+        - check if minimum_delivery_days is not higher than maximum_delivery_days
+        - check if minimum_delivery_days and maximum_delivery_days are positive values
+        """
+        min_delivery_days = cleaned_input.get("minimum_delivery_days")
+        max_delivery_days = cleaned_input.get("maximum_delivery_days")
+
+        if not min_delivery_days and not max_delivery_days:
+            return
+
+        error_occurred = False
+        if min_delivery_days and min_delivery_days < 0:
+            errors["minimum_delivery_days"] = ValidationError(
+                "Minimum delivery days must be positive.",
+                code=ShippingErrorCode.INVALID.value,
+            )
+            error_occurred = True
+        if max_delivery_days and max_delivery_days < 0:
+            errors["maximum_delivery_days"] = ValidationError(
+                "Maximum delivery days must be positive.",
+                code=ShippingErrorCode.INVALID.value,
+            )
+            error_occurred = True
+
+        if error_occurred:
+            return
+
+        if (min_delivery_days and max_delivery_days) or (
```

Comment:
Not sure, but isn't something that we can do in a single `if` condition?
```
min_delivery_days = min_delivery_days or instance.min_delivery_days
max_delivery_days = max_delivery_days or instance. max_delivery_days
if min_delivery_days > max_delivery_days:
    raise Error
```

Reply:
I did it in this way first, but then I changed it as I didn't have information which field was provided in the input.

Reply:
But I can do the check when the condition pass. Maybe it'll increase readability.